.travis.yml

```diff
@@ -45,6 +45,7 @@ install:
   - ./gradlew clean
 
 before_script:
+  - cp travis-google-services.json app/google-services.json
```

Comment:
-1.  Instead of copying the file we should be using a build flavor.  PR incoming